server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -180,7 +180,6 @@ static TransportVersion def(int id) {
     public static final TransportVersion MAX_OPERATION_SIZE_REJECTIONS_ADDED = def(9_024_0_00);
     public static final TransportVersion RETRY_ILM_ASYNC_ACTION_REQUIRE_ERROR = def(9_025_0_00);
     public static final TransportVersion ESQL_SERIALIZE_BLOCK_TYPE_CODE = def(9_026_0_00);
-    public static final TransportVersion ESQL_THREAD_NAME_IN_DRIVER_PROFILE = def(9_027_0_00);
```

Comment:
Erm, since the original PR had a TV change, we need to double check if this is actually safe to revert without breaking Serverless.